packages/block-editor/src/hooks/duotone.js

```diff
@@ -125,12 +125,14 @@ function DuotonePanel( { attributes, setAttributes } ) {
 
 	const duotonePalette = useSetting( 'color.duotone' );
 	const colorPalette = useSetting( 'color.palette' );
+	const disableCustomColors = ! useSetting( 'color.custom' );
```

Comment:
Could we inverse this? It reads more nicely without the double negative:
```jsx
const customColors = useSetting( 'color.custom' ); // or enableCustomColors
///...
{ customColors && (
    <CustomDuotoneBar value={ value } onChange={ onChange } />
}
```

Reply:
I was keeping things consistent with how [`PanelColorGradientSettings`](https://github.com/WordPress/gutenberg/blob/896ed84a811addb93fa9ab1db52f5c9a61c80641/packages/block-editor/src/components/colors-gradients/panel-color-gradient-settings.js#L83) works, but keeping that consistency probably doesn't matter that much

Reply:
Oh interesting, this one does personally read a bit awkwardly, but It's non-blocking if we're intentionally following this pattern.